starter/cloudrun-graphql/src/test/helloWorld.test.ts

```diff
@@ -0,0 +1,7 @@
+import assert = require('assert')
+
+describe('Hello world', () => {
+  it('Works', () => {
+    assert(true)
```

Comment:
We can even add a simple test for the graphql query instead of the assert, so we can make sure the app is running smoothly

Reply:
Updated ✅

Reply:
Good job, thanks